reporting/api/src/main/java/org/jboss/windup/reporting/model/ClassificationModel.java

```diff
@@ -97,4 +104,12 @@
     @SetInProperties(propertyPrefix = "tag")
     Set<String> getTags();
 
+    abstract class Impl implements ClassificationModel, JavaHandlerContext<Vertex>
```

Comment:
It _should_ work correctly. I think the only issue that we've had has had to do with trying to access overridden methods from inside the $Impl itself. There are workarounds, though.